src/documentation/ntfs2gtfs.md

```diff
@@ -0,0 +1,147 @@
+# NTFS to GTFS conversion
+## Introduction
+This document describes how a [NTFS feed](https://github.com/CanalTP/navitia/blob/dev/documentation/ntfs/) is transformed into a [GTFS feed ](https://developers.google.com/transit/gtfs/reference/) in Navitia Transit Model.
+
+The resulting GTFS feed is composed of the following objects:
+- [agency](#agency.txt)
+- [routes](#routes.txt)
+- [stops](#stops.txt)
+- [trips](#trips.txt)
+- [stop_times](#stop_times.txt)
+- [calendar_dates](#calendar_dates.txt)
+- [transfers](#transfers.txt)
+- [shapes](#shapes)
+- [stop_extensions](#stop_extensions.txt): additional information providing the complementary stop codes used in external systems.
+
+## Mapping between NTFS and GTFS objects
+#### agency.txt
+
+GTFS field | Required | NTFS file | NTFS field | Note
+--- | --- | --- | --- | ---
+agency_id | yes | networks.txt | network_id | 
+agency_name | yes | networks.txt | network_name | 
+agency_url | yes | networks.txt | network_url | 
+agency_timezone | yes | networks.txt | network_timezone | `Europe/Paris` if the value is not provided.
+agency_lang | no | networks.txt | network_lang | 
+agency_phone | no | networks.txt | network_phone | 
+
+#### routes.txt
+
+GTFS field | Required | NTFS file | NTFS field | Note
+--- | --- | --- | --- | ---
+route_id | yes | lines.txt | line_id | 
+agency_id | no | lines.txt | network_id | (link to the [agency.txt](#agency.txt) file)
+route_short_name | yes | | lines.txt | line_code | 
```

Comment:
There is an empty cell in `NTFS file` : 
`route_short_name | yes | lines.txt | line_code | `